satpy/readers/satpy_cf_nc.py

```diff
@@ -278,7 +280,7 @@ def get_dataset(self, ds_id, ds_info):
         logger.debug("Getting data for: %s", ds_id['name'])
         nc = xr.open_dataset(self.filename, engine=self.engine,
                              chunks={'y': CHUNK_SIZE, 'x': CHUNK_SIZE})
-        name = ds_info.get('orig_name', ds_id['name'])
+        name = ds_info.get('nc_store_name', ds_id['name'])
         file_key = ds_info.get('file_key', name)
         data = nc[file_key]
         if name != ds_info['name']:
```

Comment:
I think it should be `if name != ds_id['name']`, right?

Reply:
Yes of course. Thanks for pointing this out.